### PR TITLE
[SDESK-4310] [SDESK-4313] Embed and style fixes

### PR DESCRIPTION
**Pull request by fritzSF**

### CODE REVIEW

File: styles/sass/archive-preview.scss [hunk 1]
```diff
@@ -384,16 +361,27 @@
     }
 }
 
+.workspace .main-section .content-item-preview {
+    --embed-width: 52rem;
+}
+.workspace.authoring .main-section .content-item-preview {
+    --embed-width: 35.2rem;
+}
+.workspace .main-section.with-preview [sd-item-preview].sd-preview--overlay .content-item-preview {
```
Comment (tomaskikutis) [inline styles/sass/archive-preview.scss]: Don't use names of directives in selectors like `[sd-item-preview]`.

It looks like we aren't using CSS variables in the codebase yet, can we stick with sass variables?

The entire matching looks fragile in a way that it depends on a lot of things. Doesn't it work setting width to 100% and letting it adjust to the parent container?

Comment (fritzSF) [reply styles/sass/archive-preview.scss]: It doesn't work that's why it is added. CSS variables are different than sass variables as they are not compiled and they come handy in some cases. We are using them in the framework recently.

Comment (fritzSF) [reply styles/sass/archive-preview.scss]: As for the [sd-item-preview] selector, I had to use what I have. This part is a bit messy but this selector is used also on other places.

Comment (tomaskikutis) [reply styles/sass/archive-preview.scss]: Petr mentioned there's a script for controlling embed dimensions. Could you then remove CSS related to width/iframe?

**Pull request merged**